lib/flashcard_runner.rb

```diff
@@ -0,0 +1,15 @@
+require './card'
+require './turn'
+require './deck'
+require './round'
+
+card_1 = Card.new("What is 5 + 5", "10", :STEM)
+card_2 = Card.new("What is Rachel's favorite animal", "Red Panda", :"Turing_Staff")
+card_3 = Card.new("What is Mike's middle name", "Nobody knows", :"Turing_Staff")
+card_4 = Card.new("What cardboard cutout lives at Turing", "Justin Bieber", :"Pop_Culture")
```

Comment:
Same - symbols do not need quotes, they should have underscores like you have, but should be right up against the colon symbol (e.g. `:pop_culture`)